config.yaml

```diff
@@ -32,6 +32,14 @@ backend:
 #   tls:
 #     enabled: false
 #     insecure_skip_verify: false
+# redis_sentinel:
+#   sentinel_addrs: [ "127.0.0.1:26379", "127.0.0.1:26380", "127.0.0.1:26381" ]
```

Comment:
Even if we consolidate `redis-sentinel.go` and `redis.go`, I think we can keep the configuration logic as is: one for redis sentinel, and the other for redis cluster. I agree with the current approach here, let's just use the same adapter (`redis.go`) for both regular `redis` or `redis_sentinel`.